packages/database/src/modelClasses/Report.js

```diff
@@ -9,6 +9,16 @@ import { TYPES } from '../types';
 
 export class ReportType extends DatabaseType {
   static databaseType = TYPES.REPORT;
+
+  /**
+   * @returns {Promise<string>} name of permission group
+   */
+  async permissionGroupName() {
+    const permissionGroup = await this.otherModels.permissionGroup.findById(
+      this.permission_group_id,
+    );
+    return permissionGroup.name;
+  }
```

Comment:
Did this so that report-server no longer needed to be aware of the `PermissionGroup` model (and because its just simpler)